README.rst

```diff
@@ -0,0 +1,70 @@
+Image Modal XBlock
+==================
+A fullscreen image modal XBlock.
+
+This package is meant to be installed within the OpenEdX platform.
+
+.. image:: ./docs/lms-view-normal.png
```

Comment:
I don't really like the idea of keeping media assets directly in the repo;
we'll end up w/ a repo >100MB where 99% of the contents are screenshots and screencasts.

Thoughts about alternative storage?
I'd lean toward S3..

Reply:
Maybe we can switch to using JPGs and also crush them a little? should bring that down to ~250K

Reply:
So for v1, I ended up uploading to S3.

As it were, it appears that Github MITMs the URLs, even though they're absolute paths,
and rewrites the paths to go through their CDN. That CDN apparently won't serve images over a certain threshold (~6-7MB?).

So I shrunk the large PNGs by resizing them to to ~1600px (scale preserved).

I still don't have a solution for the GIF;
it's ~25MB